po/fr.po

```diff
@@ -1891,14 +1875,12 @@ msgid "Migrating configs and data from ~/.newsbeuter/..."
 msgstr ""
 
 #: rust/libnewsboat/src/configpaths.rs:214
-#, fuzzy
 msgid "Aborting migration because mkdir on `%s' failed: %s"
-msgstr "Erreur : l'ouverture du fichier de cache `%s' a échoué : %s"
+msgstr "Annulation de la migration car mkdir sur `%s' a échoué : %s"
 
 #: rust/libnewsboat/src/filterparser.rs:63
-#, fuzzy
 msgid "attribute name"
-msgstr "attribut non trouvé"
+msgstr "nom de l'attribut attribut"
```

Comment:
Repetition of "attribut" looks suspicious, is this correct? (I don't know any French.)